dFusion/dFusion.rst

```diff
@@ -6,60 +6,60 @@ dFusion - Decentralized Scalable Onchain Exchange
 
 A specification developed by Gnosis.
 
-The following specification uses the snark application (snapp) onchain scaling approach, in order to build a scalable fully decentralized exchange with decentralized order matching. 
-Scalability is achieved by storing only hashed information and allowing snarks to manipulate these through predefined logical gates `link <https://ethresear.ch/t/on-chain-scaling-to-potentially-500-tx-sec-through-mass-tx-validation/3477>`_.
+The following specification describes a scalable fully decentralized exchange with decentralized order matching. 
+Scalability is achieved by storing only hashed information onchain and allowing any bonded party to propose state-transitions.
+Predefined on-chain verification smart contracts allow any client to generate fraud proofs and thereby revert invalid state transitions.
+
 Orders are matched in a batch auction with an arbitrage-free price clearing technique developed by Gnosis: `Uniform Clearing Prices <https://github.com/gnosis/dex-research/blob/master/BatchAuctionOptimization/batchauctions.pdf>`_.
 
 Summary
 =======
 
-The envisioned exchange will enable **K** accounts to trade via limit orders between **T** predefined ERC20 tokens.
-Limit orders are collected and matched in batches, with each batch containing up to **M** orders. 
+The envisioned exchange will enable **K** (in the magnitue of 2**24) accounts to trade via limit orders between **T** (in the magnitude of 2**10) predefined ERC20 tokens.
+Limit orders are collected onchain in an orderstream and subsets of these orders are matched in batches, with each batch containing up to **M** (in the magnitude of 2**10) orders. 
```

Comment:
```suggestion
Limit orders are collected on-chain in an order-stream and subsets of these orders are matched in batches, with each batch containing up to **M** (in the magnitude of 2^10) orders. 
```

Reply:
I change the exponent here, because ** is used to denote the beginning of a bold tesxt environment. Should probably be changed everywhere.